cfme/tests/test_utilization_metrics.py

```diff
@@ -135,6 +135,7 @@ def query_metric_db(appliance, provider, metric, vm_name=None, host_name=None):
         metrics_tbl.id.in_(provs.subquery()))
 
 
+@pytest.mark.rhv1
```

Comment:
I would not consider metrics sanity

Reply:
It is a buggy area. We had several bugs, some of them were customer bugs AFAIR. I shall move the metric tests to T2.

Reply:
I agree with T2. IMHO the fact that it is generally buggy/bug free should not be considered when choosing teir

Reply:
OK